nexus/src/authz/omicron.polar

```diff
@@ -160,17 +160,11 @@ has_relation(fleet: Fleet, "parent_fleet", silo: Silo)
 #
 # It's unclear what else would break if users couldn't see their own Silo.
 has_permission(actor: AuthenticatedActor, "read", silo: Silo)
-	# TODO actor.silo is *not* a list, so `in` is incorrect here, but if you
```

Comment:
I removed this comment and the one at L171 because this behavior is documented:
https://docs.osohq.com/rust/reference/polar/classes.html#options

What wasn't obvious to me from the documentation was whether `x in y` returns true if `y` was _any_ `Some` value or only if it was `Some(x)`.  I've tested it, and it's only if `y` is `Some(x)` (which of course makes much more sense).